Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM python:3.6-alpine
```

Comment:
Went looking at pyup to see how they handle Pipfile and I found this interesting article around creating an entrypoint that activates the pipenv virtualenv at entry https://pyup.io/posts/pipfiles-and-docker/

Reply:
Good info, but unfortunately I don't think it really buys us anything, especially since we actually have to run the container in privileged mode anyway.

Reply:
Right, I hadn't made it all the way through things when I commented here.